### DISCUSSION TITLE
Upgraded cotton from 1.0.12 to 1.2.0, having issues

### DISCUSSION THREAD

**Discussion by ra-dave** [wrabit/django-cotton#200] [General]
I noticed as soon as I upgraded to 1.2.0 that some cotton components that had been commented out were now attempting to be rendered but were not rendered correctly whereas with 1.0.12 they had no effect on the page (as expected).  I assume this has something to do with the various parsing changes that were made since 1.0.12.  That alone is not a big deal but I'm also having problems with every page that uses tabs/tab cotton components based on the sample provided on the cotton samples pages, such as this error:

Invalid block tag on line 122: 'endc'. Did you forget to register or load this tag?

Was there a change to setup or configuration or something that I am missing?

**Comment by wrabit** [comment]
Hi @ra-dave difficult to know what's going on without a reproducible demo. I cannot see anything significant that could have caused this from 1.0.12 -> 1.2.0. If you're able to isolate the issue further and provide how I can reproduce, I'll be more than happy to take a look.

**Comment by danhayden** [comment]
I also started getting the error `Invalid block tag on line ...: 'endc'` after updating to 1.2.0. After looking through all my templates I noticed a template error that had failed silently before 1.2.0 but was breaking after the update.

In my case I had accidently closed the opening tag for my component like this:

```
<c-nav.item :href="page_url"/>
  {{ page_title }}
</c-nav.item>
```

Removing the `/` from the end of the opening tag resolved the issue.

```
<c-nav.item :href="page_url">
  {{ page_title }}
</c-nav.item>
```

Hope this helps you resolve your issue.

**Comment by ra-dave** [comment]
@danhayden Thanks for the tip.  In one case I had an extra closing tag and, like in your case, that wasn't causing an issue until 1.2.0.

I'm still trying to find the cause of my other issues.  Nothing jumping out to me yet, other than the fact that these have nested components.  Might be a different issue though since I'm not getting that error, my components just aren't working.

**Comment by ra-dave** [comment]
Finally found the cause of my other issues, it was a missing div close in the slot content for one of my tab components.
